server/aws-lsp-codewhisperer/src/language-server/chat/chatSessionService.ts

```diff
@@ -67,6 +67,10 @@ export class ChatSessionService {
         return this.#toolUseLookup
     }
 
+    public set toolUseLookup(toolUseLookup) {
+        this.#toolUseLookup = toolUseLookup
```

Comment:
is this being used?

Reply:
it looks like its just used for the tests, I think it should also have a type here though

Reply:
Yeah it was for the tests, and it's actually able to infer the correct type I think because of the setter pattern